src/index.js

```diff
@@ -1,82 +1,23 @@
 import './style.css';
 import './common.css';
+import fillCategories from './fillCategories';
+import renderFooter from './renderFooter';
+import getDishesFromCategory from './getDishesFromCategory';
 
-const foodDishes = document.getElementById('foodDishes');
-const arrayOfDishes = [
-  {
-    idMeal: '52772',
-    strMeal: 'Teriyaki Chicken Casserole',
-    strMealThumb:
-      'https://www.themealdb.com/images/media/meals/wvpsxx1468256321.jpg',
-    strTags: ['Meat', 'Casserole'],
-  },
-  {
-    idMeal: '52773',
-    strMeal: 'Honey Teriyaki Salmon',
-    strMealThumb:
-      'https://www.themealdb.com//images//media//meals//xxyupu1468262513.jpg',
-    strTags: ['Meat', 'Casserole'],
-  },
-  {
-    idMeal: '52774',
-    strMeal: 'Pad See Ew',
-    strMealThumb:
-      'https://www.themealdb.com//images//media//meals//uuuspp1468263334.jpg',
-    strTags: ['Meat', 'Casserole'],
-  },
-  {
-    idMeal: '52775',
-    strMeal: 'Vegan Lasagna',
-    strMealThumb:
-      'https://www.themealdb.com//images//media//meals//rvxxuy1468312893.jpg',
-    strTags: ['Meat', 'Casserole'],
-  },
-  {
-    idMeal: '52776',
-    strMeal: 'Chocolate Gateau',
-    strMealThumb:
-      'https://www.themealdb.com//images//media//meals//tqtywx1468317395.jpg',
-    strTags: ['Meat', 'Casserole'],
-  },
-  {
-    idMeal: '52777',
-    strMeal: 'Mediterranean Pasta Salad',
-    strMealThumb:
-      'https://www.themealdb.com//images//media//meals//wvqpwt1468339226.jpg',
-    strTags: ['Meat', 'Casserole'],
-  },
-];
+let categories;
 
-for (let x = 0; x < arrayOfDishes.length; x += 1) {
-  foodDishes.innerHTML += `<div class="card division flex-card dflex-row">
-      <div class="align_images">
-        <img
-          src="${arrayOfDishes[x].strMealThumb}"
-          width="554"
-          class="imagen_proyectos"
-          alt=""
-        />
-      </div>
-      <div class="text_child">
-          <h2>${arrayOfDishes[x].strMeal}</h2>
-          
-        <div class="dflex-row">
-          <span class="gray_text">• ${arrayOfDishes[x].strTags[0]} </span>
-          <span class="gray_text">• ${arrayOfDishes[x].strTags[1]}</span>
-        </div>
-          <button type="button" onclick="toggleModal(${x})" class="SeeCommentsButton" id=${arrayOfDishes[x].idMeal}>
-            <span>Comments </span>
-          </button>
-        </div>
-    </div>`;
-}
-const renderFooter = () => {
-  const footer = document.createElement('footer');
-  const footerContent = 'Created by Microverse under CC license';
+window.onload = async () => {
+  const loading = document.getElementById('loading');
+  loading.style.display = 'flex';
+  categories = await fetch('https://www.themealdb.com/api/json/v1/1/categories.php');
 
-  footer.classList.add('footer');
-  footer.innerHTML = `<p class="footer__content">${footerContent}</p>`;
-  document.body.appendChild(footer);
-};
+  categories = await categories.json();
+  categories = categories.categories;
+  console.log(categories);
```

Comment:
It floats from us to the console 😄, But don't worry about it we could change it later on, we can make a refactoring stage so we are not holding back for small things :), We will also rename some of the variables here when we get to `refactoring codes` but as the functionality it's perfect. ❤️